pkg/scheduler_plugin/plugin.go

```diff
@@ -60,9 +60,7 @@ func (p *KubeThrottler) Name() string {
 }
 
 // NewPlugin initializes a new plugin and returns it.
-func NewPlugin(configuration runtime.Object, fh framework.Handle) (framework.Plugin, error) {
-	ctx := context.TODO()
-
+func NewPlugin(ctx context.Context, configuration runtime.Object, fh framework.Handle) (framework.Plugin, error) {
```

Comment:
I wonder why we can change this signature?

Reply:
We have to, because of the change in PluginFuctory type -
https://github.com/kubernetes/kubernetes/blob/9791f0d1f39f3f1e0796add7833c1059325d5098/pkg/scheduler/framework/runtime/registry.go#L31